### ISSUE TITLE
Install/Launch Issue: CrossMgr_Setup_x64_v3_0_46-20200115141349

### ISSUE THREAD

**Issue by ChainGangEvents** [esitarski/CrossMgr#42]
I've downloaded the Windows version of the latest release and install on my pc, but when I tried to run the application, it does not want to launch. I then re-installed an earlier version (CrossMgr_Setup_v3_0_41) and it launch without trouble?

I am new to GitHub and might be doing something wrong with the download and installation.

**Comment by mbuckaway** [comment]
I just downloaded the Windows setup for v3.0.46-20200115141349. Ran the installer, clicked on the More Info button to get around the "virus wanring" and clicked Run Anyways, Installed it on my Windows 10 system, ran CrossMgr and it worked correctly.

Can I ask you outline the system you are running it on? PC Brand, amount of ram, processor, version of windows. Provide as much detail as possible.

Thanks
Mark

**Comment by mbuckaway** [comment]
The easiest way to get the info I'm looking for is to do the following:

- Open a windows command prompt window (Right Click on Start, click run, type cmd.exe)
- Run the following command:
   - systeminfo /FO LIST > systeminfo.txt
- Attach the systeminfo.txt to the github ticket (drag and drop the file onto the Github ticket entry box).

Thanks,
Mark

**Comment by ChainGangEvents** [comment]
[systeminfo.txt](https://github.com/esitarski/CrossMgr/files/4084546/systeminfo.txt)

**Comment by ChainGangEvents** [comment]
Hi Mark,

Please see above.

Thanks
Juan

**Comment by mbuckaway** [comment]
Can you clarify what " it does not want to launch" means exactly? What do you see on the screen?

Secondly, if there a reason your system is set to British as a locale (en-gb) and a en-us keyboard?

Thanks

**Comment by ChainGangEvents** [comment]
Hi Mark,

I found the problem / solution. To be able to use the GitHub new release, one have to un-install the previous version first. By just running the installer from the download, it does not over-write the application launch file in the "CrossMgr" folder. I've done it now and it seems to work. The reason for the different system set-ups is an old habit (use to work in the UK for a USA company - got use to the settings) Thank you for the help. 

Let me know if you want me to assist with testing. 

Thanks
Juan

**Comment by mbuckaway** [comment]
Thanks for the update. This is a bug.

When creating the installer update code, I wondered about whether not removing the old version first would cause an issue. Most installers do that by default. Obviously, this will require some work to make sure the user never overwrites an existing install.

Mark

**Comment by JonRosenSystems** [comment]
It would be useful to add a prompt during installation to allow the user to remove prior versions.

**Comment by mbuckaway** [comment]
@JonRosenSystems Yes. When I went through the Windows build, I was surprised the default installer didn't do that....the windows installer app is free software and has a 100 options. The wonders of "free software". Most windows installer programs detect if the current app is installed, and if so, either tell the user to uninstall the old version or run the uninstaller automatically.

Like I said, this is a bug. This will have to be updated.

For the time being, PLEASE un-install the windows version first or use the Linux/Mac version which doesn't have this problem.

**Comment by JonRosenSystems** [comment]
Thanks for the reply. I un-installed CrossMgr and SeriesMgr and the re-install was successful, with the program starting. 
On Tuesday, January 21, 2020 1:13 PM, Mark Buckaway wrote:

Date: Tue, 21 Jan 2020 10:13:08 -0800
From: Mark Buckaway
To: esitarski/CrossMgr <CrossMgr@noreply.github.com>
cc: JonRosenSystems <jrosen@jonrosensystems.com>, Mention <mention@noreply.github.com>
Subject: Re: [esitarski/CrossMgr] Install/Launch Issue: CrossMgr_Setup_x64_v3_0_46-20200115141349 (#42)
 

@JonRosenSystems Yes. When I went through the Windows build, I was surprised the default installer didn't do that....the windows installer app is free software and has a 100 options. The wonders of "free software". Most windows installer programs detect if the current app is installed, and if so, either tell the user to uninstall the old version or run the uninstaller automatically.

Like I said, this is a bug. This will have to be updated.

For the time being, PLEASE un-install the windows version first or use the Linux/Mac version which doesn't have this problem.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
